content/ch-quantum-hardware/measurement-error-mitigation.ipynb

```diff
@@ -262,10 +262,6 @@
     "Unfortunately, this is the exact opposite of what we need. Instead of a way to transform ideal counts data into noisy data, we need a way to transform noisy data into ideal data. In linear algebra, we do this for a matrix $M$ by finding the inverse matrix $M^{-1}$,\n",
     "\n",
     "$$C_{ideal} = M^{-1} C_{noisy}.$$\n",
```

Comment:
```suggestion
     "$$C_{ideal} = M^{-1} C_{noisy}.$$\n"
```
The build fails, because the notebook is no valid JSON anymore. You'll have to remove the comma here.

Reply:
This change does fix the notebook, @jahanclaes please commit this change before I can merge it